docs/source/release_notes.rst

```diff
@@ -21,11 +33,7 @@ Release Notes
         * Fixed cost benefit matrix demo formatting :pr:`2990`
         * Update ReadMe.md with new badge links and updated installation instructions for conda :pr:`2998`
         * Added more comprehensive doctests :pr:`3002`
-    * Testing Changes
```

Comment:
We normally keep sections even if they're empty right? Not a big deal at all though!

Reply:
I've been sniping them!